pkg/addons/embeddedclusteroperator/upgrade.go

```diff
@@ -15,7 +15,7 @@ func (e *EmbeddedClusterOperator) Upgrade(ctx context.Context, kcli client.Clien
 		return errors.Wrap(err, "check if release exists")
 	}
 	if !exists {
-		slog.Info("Release not found, installing", "release", releaseName, "namespace", namespace)
+		logrus.Debugf("Release not found, installing release %s in namespace %s", releaseName, namespace)
```

Comment:
why change to debug? debug logs aren't easily accessibly in the pods like the cli no?

Reply:
It is this way in all other addons
![Screenshot 2025-03-26 at 8 05 22 AM](https://github.com/user-attachments/assets/833c2b86-8c94-41e7-9d64-d6cb48ac2cf8)